### PR TITLE
Fixbug i18n #2008

### PR DESCRIPTION
**Pull request by wenhemin**
1.api return message add chinese content
2.fix naming error

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/DataSourceController.java [hunk 1]
```diff
@@ -391,8 +391,8 @@ public Result verifyDataSourceName(@ApiIgnore @RequestAttribute(value = Constant
         try {
             return dataSourceService.verifyDataSourceName(loginUser, name);
         } catch (Exception e) {
-            logger.error(VERFIY_DATASOURCE_NAME_FAILURE.getMsg(),e);
-            return error(VERFIY_DATASOURCE_NAME_FAILURE.getCode(), VERFIY_DATASOURCE_NAME_FAILURE.getMsg());
+            logger.error(VERIFY_DATASOURCE_NAME_FAILURE.getMsg(),e);
```
Comment (Jave-Chen) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/DataSourceController.java]: Space after comma, ie `logger.error(VERIFY_DATASOURCE_NAME_FAILURE.getMsg(), e);`

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java [hunk 2]
```diff
@@ -16,251 +16,259 @@
...
+    USER_LIST_ERROR(10099,"user list error", "查询用户列表错误"),
+    VERIFY_USERNAME_ERROR(10100,"verify username error", "用户名验证错误"),
+    UNAUTHORIZED_USER_ERROR(10101,"unauthorized user error", "查询未授权用户错误"),
+    AUTHORIZED_USER_ERROR(10102,"authorized user error", "查询授权用户错误"),
+    QUERY_TASK_INSTANCE_LOG_ERROR(10103,"view task instance log error", "查询任务实例日志错误"),
+    DOWNLOAD_TASK_INSTANCE_LOG_FILE_ERROR(10104,"download task instance log file error", "下载任务日志文件错误"),
+    CREATE_PROCESS_DEFINITION(10105,"create process definition", "创建工作流错误"),
+    VERIFY_PROCESS_DEFINITION_NAME_UNIQUE_ERROR(10106,"verify process definition name unique error", "工作流名称已存在"),
+    UPDATE_PROCESS_DEFINITION_ERROR(10107,"update process definition error", "更新工作流定义错误"),
+    RELEASE_PROCESS_DEFINITION_ERROR(10108,"release process definition error", "上下线工作流错误"),
```
Comment (Technoboy-) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java]: 上线工作流错误

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java [hunk 3]
```diff
@@ -16,251 +16,259 @@
...
+    DELETE_PROCESS_INSTANCE_BY_ID_ERROR(10117,"delete process instance by id error", "删除工作流实例错误"),
+    QUERY_SUB_PROCESS_INSTANCE_DETAIL_INFO_BY_TASK_ID_ERROR(10118,"query sub process instance detail info by task id error", "查询子流程任务实例错误"),
+    QUERY_PARENT_PROCESS_INSTANCE_DETAIL_INFO_BY_SUB_PROCESS_INSTANCE_ID_ERROR(10119,"query parent process instance detail info by sub process instance id error", "查询子流程该工作流实例错误"),
+    QUERY_PROCESS_INSTANCE_ALL_VARIABLES_ERROR(10120,"query process instance all variables error", "查询工作流自定义变量信息错误"),
+    ENCAPSULATION_PROCESS_INSTANCE_GANTT_STRUCTURE_ERROR(10121,"encapsulation process instance gantt structure error", "查询工作流实例甘特图数据错误"),
+    QUERY_PROCCESS_DEFINITION_LIST_PAGING_ERROR(10122,"query proccess definition list paging error", "分页查询工作流定义列表错误"),
+    SIGN_OUT_ERROR(10123,"sign out error", "退出错误"),
+    TENANT_CODE_HAS_ALREADY_EXISTS(10124,"tenant code has already exists", "租户编码已存在"),
+    IP_IS_EMPTY(10125,"ip is empty", "IP地址不能为空"),
+    SCHEDULE_CRON_REALEASE_NEED_NOT_CHANGE(10126, "schedule release is already {0}", "调度配置上下线错误[{0}]"),
```
Comment (Technoboy-) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java]: 调度配置上线错误

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java [hunk 4]
```diff
@@ -16,251 +16,259 @@
...
+    UPDATE_ACCESS_TOKEN_ERROR(70013,"update access token error", "更新访问token错误"),
+    DELETE_ACCESS_TOKEN_ERROR(70014,"delete access token error", "删除访问token错误"),
+    ACCESS_TOKEN_NOT_EXIST(70015, "access token not exist", "访问token不存在"),
 
 
-    COMMAND_STATE_COUNT_ERROR(80001,"task instance state count error"),
+    COMMAND_STATE_COUNT_ERROR(80001,"task instance state count error", "查询各状态任务实例数错误"),
 
-    QUEUE_COUNT_ERROR(90001,"queue count error"),
+    QUEUE_COUNT_ERROR(90001,"queue count error", "查询数据错误"),
```
Comment (Technoboy-) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java]: 队列

**Comment by wenhemin** [comment]
> Does the front-end interface need to be modified too?

No need to modify

**Comment by wenhemin** [comment]
This PR is implemented based on the current code, Only English and Chinese messages is supported, I think it can meet the current demand.
Next step, will extract the message into the i18n properties file, this change is relatively large.

**Pull request merged**